src/storage.k.md

```diff
@@ -44,12 +44,6 @@ We state here that no such collisions are possible, allowing for simplification.
 rule keccakIntList(A B) ==K 75506153327051474587906755573858019282972751592871715030499431892688993766217 => false
 requires A =/=Int 0
 
-rule keccakIntList(A B) ==K 75506153327051474587906755573858019282972751592871715030499431892688993766217 => false
-requires 0 =/=Int A
-
-rule 75506153327051474587906755573858019282972751592871715030499431892688993766217 ==K keccakIntList(A B) => false
-requires A =/=Int 0
-
 rule 78338746147236970124700731725183845421594913511827187288591969170390706184117 ==K keccakIntList(A B) => false
```

Comment:
What is this number?

Reply:
this is the slot for `balanceOf[0]`. both the two lines you deleted above are required afaik. They flip the order in the requires. I really had to add all four rules here or I would sometimes get stupid non deterministic failures.